packages/frontend/src/hooks/useSpaces.ts

```diff
@@ -26,6 +32,25 @@ export const useSpaces = (
     );
 };
 
+const getSpaceSummaries = async (projectUuid: string) => {
+    return lightdashApi<SpaceSummary[]>({
+        url: `/projects/${projectUuid}/spaces`,
+        method: 'GET',
+        body: undefined,
+    });
+};
+
+export const useSpaceSummaries = (
+    projectUuid: string,
+    queryOptions?: UseQueryOptions<SpaceSummary[], ApiError>,
+) => {
+    return useQuery<SpaceSummary[], ApiError>(
+        ['projects', projectUuid, 'spaces'],
```

Comment:
do we need to refetch spaces if queryOptions changes ?

Reply:
Ah yes we should!

Reply:
I'm not sure actually, the only time it is used is to change the `onSuccess` - so potentially it could be useful. Are you saying we should manually trigger a refetch if it happens? I think right now it won't because the data is already cached